examples/delete.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+require  dirname(__FILE__) . '/../src/sajari.php';
```

Comment:
@tbillington for all the examples, once you put the classes/interfaces into their own files, you won't be able to just `require` one file (because there will be too many). In the v9 SDK it was annoying using composer to require Sajari for my examples, so you're thinking is right to include the *real source* so maybe you can look at how some other repos require "themselves" in their examples. Let me know if my comments are not clear here.